packages/build/src/core/build.ts

```diff
@@ -124,6 +127,20 @@ const tExecBuild = async function ({
     timers,
     quiet,
   })
+
+  if (featureFlags['build_automatic_runtime'] && framework) {
+    const runtime = supportedRuntimes[framework]
+
+    if (runtime !== undefined) {
+      const skip = childEnv[runtime.skipFlag] === 'true'
+      const installed = netlifyConfig.plugins.map((plugin) => plugin.package).includes(runtime.package)
```

Comment:
It's not a big deal, but I think it's unnecessary to traverse the array twice.

```suggestion
      const installed = netlifyConfig.plugins.some((plugin) => plugin.package === runtime.package)
```